content/articles/how-edge-artificial-intelligence-is-revolutionizing-mobile-application-development/index.md

```diff
@@ -0,0 +1,85 @@
+### How Edge Artificial Intelligence is Revolutionizing Mobile Application Development
+
+The explosive growth of artificial intelligence has influenced app development in recent years. AI and machine learning have matured to provide flexible algorithms for intuitive and seamless experiences. Developers, users, and businesses now have a whole new view of intelligent interactions within mobile apps.
+
+They have embraced the current trend in the mobile app industry dominated by the Internet of Things (IoT) and mobile computing applications. IoT and mobile devices generate loads and loads of data at the network edge. Collecting such massive amounts of data in cloud data centers comes with a fair share of challenges, such as high network bandwidth and latency usage.
+
+The solution to fully unleash the potential of big data and avoid issues associated with cloud computing is to bring AI to the network edge. This is what edge artificial intelligence does by combining AI and edge computing.
+
+This article looks at artificial intelligence at the edge, discusses how it is changing the mobile application development landscape and gives examples of intelligence edge.
+
+### Artificial intelligence at the edge
+
+#### Edge AI is driven by and IoT and big data
+
+In today's Internet of Things (IoT) era, connected devices are generating large volumes of data available for collection and analysis. Artificial intelligence systems are required to make sense of this large quantity of data in real-time.
+
+A typical edge computing environment hosts machine learning models on the cloud. In this setup, a prediction request is sent from the device to an application programming interface ([API](https://www.ibm.com/cloud/learn/api)) based on the cloud. A response to the request is sent back over the internet.
+
```

Comment:
#### Edge AI is driven by and IoT and big data

@BashiirIsla  Please move this heading to this place.